api/backend.proto

```diff
@@ -120,7 +118,7 @@ service Backend {
   // IDs passed.
   rpc AssignTickets(AssignTicketsRequest) returns (AssignTicketsResponse) {
     option (google.api.http) = {
-      post: "/v1/backend/tickets/assign"
+      post: "/v1/backend/tickets:assign"
```

Comment:
For my understanding, what is this?
I don't see it used in https://github.com/grpc-ecosystem/grpc-gateway/blob/master/examples/proto/examplepb/a_bit_of_everything.proto.